test/test_playbooks/content_view.yml

```diff
@@ -70,12 +181,6 @@
     vars:
       product_name: "Test Product 2"
       product_state: absent
-  - include: tasks/repository.yml
-    vars:
-      repository_state: absent
-  - include: tasks/product.yml
-    vars:
-      product_state: absent
```

Comment:
Because the product and repo is now in a published CCV and the modules cannot remove a CCV that is in an environment. https://github.com/theforeman/foreman-ansible-modules/blob/master/modules/katello_content_view_version.py#L31

Reply:
Sounds like a reason. So we leave a little clutter on the test instance. Do we state, that you should never record against you production environment?

Reply:
I don't believe so, but I think everyone should know not to record tests on servers they care about?

Also it still done inside the Test Organization so it *shouldn't* cause any issues really.